frontend/src/clients/Httpclient.ts

```diff
@@ -32,15 +29,11 @@ export class HttpClient {
               throw new NotFoundException(errorMessage)
             case HttpStatusCode.Forbidden:
               throw new ForbiddenException(errorMessage)
-            case HttpStatusCode.InternalServerError:
```

Comment:
ServiceUnavailable is not like InternalServerError, maybe need a session on this scenario